README.md

```diff
@@ -93,12 +93,15 @@ from labkey.api_wrapper import APIWrapper
 
 print("Create an APIWrapper")
 labkey_server = 'localhost:8080'
-project_name = 'ModuleAssayTest'  # Project folder name
+container_path = 'Tutorials/HIV Study'  # in this example, Tutorials is a project name and HIV study is a subfolder under it.
 contextPath = 'labkey'
 schema = 'core'
 table = 'Users'
-api = APIWrapper(labkey_server, project_name, contextPath, use_ssl=False)
 
+# Note: If developing against localhost with https disabled, set use_ssl=False below
+api = APIWrapper(labkey_server, container_path, contextPath)
+
+# Makes an API request to https://www.example.com/labkey/ModuleAssayTest/query-getQuery.api
```

Comment:
do you want to replace the `ModuleAssayTest` in the example URL here as well to match the container_path at line 96?

Reply:
Yeah we should definitely fix that, otherwise it will be rather confusing

Reply:
good catch, thanks guys